docs/classifier_example/classification_example3_4_features.ipynb

```diff
@@ -0,0 +1,893 @@
+       " -0.5863773622428564]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Load scaled and normalised data\n",
+    "iris_setosa_normalised, iris_versicolor_normalised = DataPlotter.load_data(max_features=4)\n",
+    "first_data_point = [feature[0] for feature in iris_setosa_normalised]\n",
+    "first_data_point"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We want to build the circuit such that:\n",
+    "\\begin{equation}\\ket{00} \\mapsto -0.3270 \\ket{00} + 0.4737 \\ket{01} - 0.5700 \\ket{10} - 0.5864 \\ket{11}\\end{equation}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a00, a01, a10, a11 = first_data_point\n",
+    "alpha1 = atan(sqrt((a10**2  + a11**2) / (a00**2  + a01**2)))\n",
+    "alpha2 = np.arctan2(a01, a00) \n",
+    "alpha3 = np.arctan2(a11, a10) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As always the first step is to set up a connection with the Quantum Inspire:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from getpass import getpass\n",
+    "from quantuminspire.credentials import load_account, get_token_authentication, get_basic_authentication\n",
+    "\n",
+    "from qiskit.validation.base import Obj\n",
+    "from qiskit.circuit import QuantumRegister, ClassicalRegister, QuantumCircuit\n",
+    "from qiskit.tools.visualization import plot_state, plot_histogram, circuit_drawer\n",
+    "from qiskit import execute\n",
+    "\n",
+    "from quantuminspire.qiskit import QI\n",
+    "from quantuminspire.api import QuantumInspireAPI\n",
+    "\n",
+    "QI_EMAIL = os.getenv('QI_EMAIL')\n",
+    "QI_PASSWORD = os.getenv('QI_PASSWORD')\n",
+    "QI_URL = os.getenv('API_URL', 'https://api.quantum-inspire.com/')\n",
+    "\n",
+    "\n",
+    "def get_authentication():\n",
+    "    \"\"\" Gets the authentication for connecting to the Quantum Inspire API.\"\"\"\n",
+    "    token = load_account()\n",
+    "    if token is not None:\n",
+    "        return get_token_authentication(token)\n",
+    "    else:\n",
+    "        if QI_EMAIL is None or QI_PASSWORD is None:\n",
+    "            print('Enter email:')\n",
+    "            email = input()\n",
+    "            print('Enter password')\n",
+    "            password = getpass()\n",
+    "        else:\n",
+    "            email, password = QI_EMAIL, QI_PASSWORD\n",
+    "        return get_basic_authentication(email, password)\n",
+    "\n",
+    "    \n",
+    "authentication = get_authentication()\n",
+    "# Temporairy alternative so that we can give our project a name:\n",
+    "QI._api = QuantumInspireAPI(QI_URL, authentication, project_name=\"Distance-based Classifier more features (NEW)\")\n",
```

Comment:
access to private member, is there no other way to name the project?

Reply:
We have [DEM-869] for this.